libraries/consul.rb

```diff
@@ -18,5 +18,23 @@ def self.setup_consul(options)
         options[:token] = options[:consul_token]
       end
     end
+
+    def self.update_backup_url(options)
+      require 'diplomat'
+      require 'diplomat/version'
```

Comment:
This line is not needed as you don't access the VERSION const

Reply:
Indeed